### PR TITLE
Modules, ints, and floats concepts

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: exercises/concept/bettys-bike-shop/.docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+All functions within that module that were exposed by it are made accessible when importing that module.
+But how they are accessed varies depending on how the module is imported.
+
+Qualified imports are the default, and accessing a function within such module (for example the `to_string` function in the `gleam/int` module) is done by prefixing the module name (`int.to_string`).
+
+```gleam
+// Import the int module
+import gleam/int
+
+pub fn main(x: Int) -> String {
```
Comment (bcpeinhardt) [inline exercises/concept/bettys-bike-shop/.docs/introduction.md:88]: Maybe name these something other than `main`? Someone might read this as "the main function in a gleam program returns a String".

File: concepts/floats/.meta/config.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "blurb": "There are two different kinds of numbers in Elixir - integers and floats. Floats are numbers with one or more digits behind the decimal separator.",
```
Comment (bcpeinhardt) [inline concepts/floats/.meta/config.json:2]: Is this supposed to say gleam? Not sure where this gets used.

File: concepts/floats/about.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+2.0 >. 1.0  // -> True
+2.0 <. 1.0  // -> False
+2.0 >=. 1.0 // -> True
+2.0 <=. 1.0 // -> False
+```
+
+Underscores can also be added to floats for clarity.
+
+```gleam
+1_000_000.0 // One million
```
Comment (bcpeinhardt) [inline concepts/floats/about.md:27]: What about after the decimal?

Comment (lpil) [reply concepts/floats/about.md:27]: The formatter won't insert underscores there at the moment

**Pull request merged**